.travis-build.sh

```diff
@@ -6,6 +6,40 @@ set -e
 
 export SHELLOPTS
 
+# Optional argument $1 is one of:
+#   cfi-tests, downstream
+# If it is omitted, this script does everything.
+export GROUP=$1
+if [[ "${GROUP}" == "" ]]; then
+  export GROUP=all
+fi
+
+SLUGOWNER=${TRAVIS_REPO_SLUG%/*}
+if [[ "$SLUGOWNER" == "" ]]; then
+  SLUGOWNER=opprop
+fi
+
+if [[ "${GROUP}" != "cfi-tests" && "${GROUP}" != "downstream" ]]; then
```

Comment:
Looks like if GROUP=all case will be considered as bad argument and exit

Reply:
Ah, good catch! Will fix this.